cmd/lifecycle/analyzer.go

```diff
@@ -207,21 +220,29 @@ func (aa analyzeArgs) analyze() (platform.AnalyzedMetadata, error) {
 	return analyzedMD, nil
 }
 
-func (a *analyzeCmd) registryImages() []string {
-	var registryImages []string
-	if a.platform06.cacheImageTag != "" {
-		registryImages = append(registryImages, a.platform06.cacheImageTag)
-	}
-	if !a.useDaemon {
-		registryImages = append(registryImages, append([]string{a.imageName, a.previousImage, a.runImageRef}, a.additionalTags...)...)
-	}
-	return registryImages
+func (a *analyzeCmd) platformAPIVersionGreaterThan06() bool {
+	return api.MustParse(a.platform.API()).Compare(api.MustParse("0.7")) >= 0
 }
 
 func (a *analyzeCmd) restoresLayerMetadata() bool {
 	return !a.platformAPIVersionGreaterThan06()
 }
 
-func (a *analyzeCmd) platformAPIVersionGreaterThan06() bool {
-	return api.MustParse(a.platform.API()).Compare(api.MustParse("0.7")) >= 0
+func (a *analyzeCmd) ReadableImages() []string {
```

Comment:
```suggestion
func (a *analyzeCmd) ReadableRegistryImages() []string {
```

Reply:
(this seems more accurate)

Reply:
Fixed in the following [commit](https://github.com/buildpacks/lifecycle/pull/646/commits/5c900400a2e89ae0d9f8a8d0c02243dec574ceba)